azure-pipelines-pr.yml

```diff
@@ -0,0 +1,42 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - '*'
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      endpoint: Planning-Inspectorate
+      name: Planning-Inspectorate/common-pipeline-templates
+      ref: refs/tags/release/3.5.0
+
+extends:
+  template: stages/wrapper_ci.yml@templates
+  parameters:
+    globalVariables:
+      - template: azure-pipelines-variables.yml@self
+    validationJobs:
+      - name: Run Linting & Tests
+        steps:
+          - template: ../steps/node_script.yml
+            parameters:
+              nodeVersion: 18
+              script: make install
+          - template: ../steps/node_script.yml
+            parameters:
+              condition: succeededOrFailed()
+              script: npm run lint
+              nodeVersion: 18
+          - template: ../steps/node_script.yml
+            parameters:
+              condition: succeededOrFailed()
+              script: npm run test
+              nodeVersion: 18
+          - template: ../steps/node_script.yml
+            parameters:
+              condition: succeededOrFailed()
+              script: npm run integration-test
```

Comment:
So previously we had integration tests only running on main
They used to be flaky so were causing us to need to rebuild to get through a PR
Did spend some time resolving that and haven't seen it fail on main for a while but worth noting we may need to take this out or spend some more time looking at them

Reply:
That's good to know, I mis-read the condition!

I checked a few runs at 'random' and they've all run OK. Let's keep it in, and review.